src-back/app/fight/fight_session.py

```diff
@@ -24,13 +26,26 @@ class FightSession:
     def __init__(self, fight_id, player_list):
         self._fight_id = fight_id
         self._players = [player for player in player_list]
-        self._players_sessions = {}
+        self._players_sessions: dict[str, WebSocket] = {}
         self._players_typing_history = {player: [] for player in player_list}
         self._scores = {player: 0 for player in player_list}
         self._words_to_find = ["word", "broccoli", "tomato", "bee", "beekeeper"]
         self._words_found = []
         self._word_best_progress = {}
-        self._game_end_epoch = int(time.time() + 60)
+        self._game_start_wait_ms = 30_000
+        self._game_duration_ms = 60_000
```

Comment:
Just wanted to make sure : We will have to make this configurable later, since the "Lobby" page dictates the duration of the game.
So this info will have to pass through one of the routes and end up here. If that is ok, you can discard this comment.

Reply:
In my opinion it is a matter of adding additional parameters to the constructor, say a settings struct if we want to keep it clean.